merkle/accumulator_test.go

```diff
@@ -194,8 +196,10 @@ func TestApplyBlock(t *testing.T) {
 	for i := range oldSpent {
 		oldSpent[i].MerkleProof = append([]types.Hash256(nil), oldSpent[i].MerkleProof...)
 	}
-	eau := acc.ApplyBlock(spent, nil)
-
+	var eau ElementApplyUpdate
+	for i := 0; i < 5; i++ {
+		eau = acc.ApplyBlock(spent, nil)
+	}
```

Comment:
this `ApplyBlock` is just updating existing leaves, not adding new ones, so I *think* calling it more than once will have no effect on the accumulator (but we might want to confirm that, just in case)